content/_includes/site/footer.njk

```diff
@@ -1,10 +1,26 @@
 {% import 'contact.macros.njk' as contact %}
 {% import 'utility.macros.njk' as utility %}
 
-<div
-  data-layout="footer">
+{%- set add_cta = page.url | addCallToAction -%}
+{%- if add_cta %}
+  {% include 'site/cta.njk' %}
+{% endif -%}
+
+<div data-layout="footer">
   {% if page.url != '/contact/' %}
     <aside aria-label="contact">
+      {%- if not add_cta %}
+        <div class="contact-form-intro">
+          <h3>Let's chat about your web project</h3>
```

Comment:
I suppose we might want to run this thru Markdown too, to get e.g. `’` instead of `'`?